src/BenchmarkDotNet/Configs/DefaultConfig.cs

```diff
@@ -87,6 +87,8 @@ public IEnumerable<IValidator> GetValidators()
 
         public TimeSpan BuildTimeout => TimeSpan.FromSeconds(120);
 
+        public WakeLockType WakeLock => WakeLockType.No;
```

Comment:
Should the default be system?

Reply:
Wait, does this require administrator privileges? If so, we can't use it by default. I don't see it called out in the docs, only in the test comments.

Reply:
The feature itself does **not** require admin privileges. Any process can create power requests.

`powercfg.exe /requests` does, so one must run the test code with elevation. That's why I put all test code in integration tests.